app/subscriber/src/features/hooks/useSubMediaGroups.ts

```diff
@@ -57,7 +50,7 @@ export const useSubMediaGroups = (
             l.options.push({
               id: s.id,
               listOption: ListOptionName.Source,
-              name: getSourceName(s),
+              name: s.name,
```

Comment:
Are you certain this is correct?  Normally Scott wants the Common Call (`shortName`) if it's available.

Reply:
Not now. We reviewed that and I changed to the regular name on the fly, and he wanted like this.
Probably because we have the Common Call now at the groups description on the results side.